src/test/rgw/rgw_multi/multisite.py

```diff
@@ -367,12 +368,16 @@ def create(self, zone, args = None, **kwargs):
         """ create the user with the given arguments """
         assert(zone.cluster)
         args = zone.zone_args() + (args or [])
+        if self.tenant:
+            args += ['--tenant', self.tenant]
```

Comment:
can you move these bits to `user_arg()` instead? doing it there will apply to any `json_command()` call